test/support/rails_app.rb

```diff
@@ -1,3 +1,5 @@
+require 'action_mailer/railtie'
```

Comment:
ref: https://github.com/rails-api/active_model_serializers/pull/1550/files#r54681004 we can probably remove action_mailer here and below